File: Clg_Placement/Java/_17.CompeteTheSkills.java
class Solution {

    public void scores(long a[], long b[]) {
        int a1 = 0;
        int b1 = 0;
        for(int i=0;i<3;i++){
            if(a[i]>b[i]) a1+=1;
            else if(a[i]<b[i]) b1+=1;
            else continue;
        }
        GFG.ca = a1;
        GFG.cb = b1;
    }
}

File: Clg_Placement/C++/Arrays/_17.CompeteTheSkills.cpp
class Solution{
    public:
    void scores(long long a[], long long b[], int &ca, int &cb)
    {
        ca = 0 , cb = 0;
        for(int i=0;i<3;i++){        //comparing three skills so directly give 3.
            if(a[i]>b[i]) ca+=1;
            else if(a[i]<b[i]) cb+=1;
            else continue;
        }
};
